Directory.Build.props

```diff
@@ -16,7 +16,7 @@
           
           DEV # Enables some developer tools.
     -->
-    <DefineConstants>$(DefineConstants);SRDX</DefineConstants>
+    <DefineConstants>$(DefineConstants);SRTE</DefineConstants>
```

Comment:
Don't change the default, because it will make the result of default build inconsistent with the README. Instead update the README to add `SRTE` as one of the possibilities there.